doc/source/operations/cloudkitty.rst

```diff
@@ -0,0 +1,125 @@
+==========
+CloudKitty
+==========
+
+Configuring in kayobe-config
+============================
+
+By default, CloudKitty uses Gnocci and Ceilometer as the collector and fetcher
+backends. Unless the system has a specific reason not to, we recommend instead
+using Prometheus as the backend for both. The following instructions explain
+how to do this
+
+Enable CloudKitty and disable InfluxDB, as we are using OpenSearch as the
+storage backend. Set the following in ``kolla.yml``:
+
+.. code-block:: console
+
+  kolla_enable_cloudkitty: true
+  # Explicitly disable influxdb as we are using OpenSearch as the CloudKitty backend
```

Comment:
Not sure if this is necessary, as long as it isn't made the backend then it should be fine, no? 
Leaving it commented out may be sufficient?

Reply:
Sure it will be fine in the sense that nothing will break. But when KA sees that CloudKitty is enabled, it will also deploy InfluxDB by default, so we'd have an InfluxDB container just sat there doing nothing.

Reply:
Really it would be good to patch KA to fix this properly, but I recall there being some complication that made this non-trivial